realm/realm-library/src/main/java/io/realm/sync/permissions/PermissionUser.java

```diff
@@ -72,4 +74,14 @@ public String getId() {
     public @Nullable  RealmResults<Role> getRoles() {
         return roles;
     }
+
+    /**
+     * The user's private role. This will be initialized to a role named for the user's
+     * identity that contains this user as its only member.
+     *
+     * @return User private {@link Role}.
+     */
+    public Role getRole() {
```

Comment:
Any reason for calling this `getRole` vs. e.g. `getPrivateRole()` which seems more descriptive?